### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by davidvgalbraith**
@demmer @VladVega

### CODE REVIEW

**Comment by VladVega** [comment]
Looks good.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Concurrency Master
+
+The Concurrency Master is a lightweight promise-based task scheduler. Initialize it with a concurrency value, then call its `add` method on functions that return [Promises](https://www.promisejs.org/). The ConcurrencyMaster will execute all the given promise-returning functions. In doing so, it will make sure that no more than its given concurrency are running in parallel.
+
+The `wait()` method returns a Promise that resolves when all of the ConcurrencyMaster's `push`ed Promises have resolved.
```
Comment (demmer) [inline README.md:5]: `add` not `push`

**Comment by demmer** [comment]
Heh -- review came a minute late.

**Comment by davidvgalbraith** [comment]
Thanks! Pushed https://github.com/juttle/concurrency-master/commit/3f95b9357e14ff2b42c9b5a101f01b7d8636752c straight to master.

**Pull request merged**